seed_tl.ps1

```diff
@@ -193,68 +238,95 @@ $loopSleep = ($setting.loopSleep)
 $closeGameConsole = ($setting.closeGameConsole)
 $closeGame = ($setting.closeGame)
 $tl = ($setting.timestamps)
-$servers = ((Invoke-webrequest -UseBasicParsing -URI "http://131.153.65.166/files/seedscript/servers.txt").Content).Split(",")
+$serversFromWeb = ((Invoke-webrequest -UseBasicParsing -URI "http://131.153.65.166/files/seedscript/servers.txt").Content).Split(",")
 $verbose = ($setting.verbose)
 $updater = ($setting.updater)
 $waitForInput = ($setting.waitForInput)
 $scheduler = ($setting.schdulerEnabled)
 $seedStart = [int]($setting.seedStart)
 $seedEnd = [int]($setting.seedEnd)
+$checkOtherServersIntervalMinutes = [int]($setting.checkOtherServersIntervalMinutes)
+$resizeWindow = [int]($setting.resizeWindow)
+$resizeAlternateMethod = [int]($setting.resizeAlternateMethod)
+
+if ($checkOtherServersIntervalMinutes -eq 0) {
```

Comment:
Do we need this?  We already force a new config file and the new config file will have this value.